CMakeLists.txt

```diff
@@ -2,30 +2,47 @@ cmake_minimum_required(VERSION 3.7.0)
 
 project(dcos-mesos-modules)
 
-set(MESOS_ROOT "../mesos" CACHE PATH "Mesos root directory.")
+include(CTest) # NOTE: BUILD_TESTING is defined by CTest.
+
+set(MESOS_ROOT "../mesos" CACHE FILEPATH "Mesos root directory.")
```

Comment:
For these variables, why FILEPATH instead of PATH?

From the cmake docs:

```
FILEPATH    Path to a file on disk.
PATH        Path to a directory on disk.
```